folly/experimental/symbolizer/Dwarf.cpp

```diff
@@ -19,7 +19,11 @@
 
 #include <type_traits>
 
-#include <dwarf.h>
+#if FOLLY_HAVE_LIBDWARF_DWARF_H
+# include <libdwarf/dwarf.h>
```

Comment:
I didn't want to add it to the header search directory for every compilation. Not that there would be anything else in there, but it's more in the style of boost/folly/etc to include its files via the name of the project. This case just happens to be messier because it historically didn't ship its headers in a subdir.

Also seems tidier in case it's installed in `/usr/local/include` or wherever the user has installed it — they don't need to explicitly add `-I/usr/local/include/libdwarf`, they can just configure with `CPPFLAGS=-I/usr/local/include`.